### PR TITLE
Use application title and user commonName in emails

### PR DESCRIPTION
**Pull request by opqdonut**
groundwork for #1366

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: src/clj/rems/poller/email.clj [hunk 1]
```diff
@@ -31,9 +32,18 @@
   (str (:public-url env) "accept-invitation?token=" token))
 
 (defn- application-id-for-email [application]
```
Comment (okahilak) [inline src/clj/rems/poller/email.clj]: Hmm, would you be willing to change the function name to reflect the addition of application title/description? (e.g., application-id-and-description-for-email or if you come up with something better...)

File: src/clj/rems/poller/email.clj [hunk 2]
```diff
@@ -31,9 +32,18 @@
...
-  (case (util/getx env :application-id-column)
-    :external-id (:application/external-id application)
-    :id (:application/id application)))
+  (str
+   (case (util/getx env :application-id-column)
+     :external-id (:application/external-id application)
+     :id (:application/id application))
```
Comment (okahilak) [inline src/clj/rems/poller/email.clj]: I'd actually like to have a comma between the id and the description -- or a similar separator, but I think comma is the least conspicuous.

File: test/clj/rems/poller/test_email.clj [hunk 3]
```diff
@@ -60,19 +60,34 @@
...
   (let [application (-> (reduce model/application-view nil events)
                         (model/enrich-with-injections {:get-workflow get-workflow
                                                        :get-catalogue-item get-catalogue-item
-                                                       :get-form-template get-nothing
+                                                       :get-form-template get-form-template
                                                        :get-license get-nothing
                                                        :get-user get-nothing
                                                        :get-users-with-role get-nothing
                                                        :get-attachments-for-application get-nothing}))]
+    (prn :APPLICATION application)
```
Comment (okahilak) [inline test/clj/rems/poller/test_email.clj]: debug code?

**Comment by okahilak** [comment]
Did you consider mentioning the change in the change log?

File: src/clj/rems/poller/email.clj (hunk 2 above)
Comment (opqdonut) [reply src/clj/rems/poller/email.clj]: fine by me

File: test/clj/rems/poller/test_email.clj (hunk 3 above)
Comment (opqdonut) [reply test/clj/rems/poller/test_email.clj]: removed

File: src/clj/rems/poller/email.clj (hunk 1 above)
Comment (opqdonut) [reply src/clj/rems/poller/email.clj]: done

**Pull request merged**